index.html

```diff
@@ -175,5 +175,21 @@ <h3 class="view">Skills <img src="img/Normal Button/Tertiary/Icons/Union.png" al
         </div>
     </section> 
     <!-- End about me -->
+    <!-- Start contact -->
+    <footer>
+        <div class="contain">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xyyvwqrd" method="POST">
+                <input type="text" required name="full-name" placeholder="full name">
+                <input type="email" required name="Email" placeholder="email">
+                <textarea name="Message" placeholder="write your message here."></textarea>
+                <button type="submit" class="normal-button">Get in touch</button>
+            </form>
```

Comment:
Kindly make sure your input fields are validated. 
- [ ] name field must accept up to 30 chars. `hint: max-length="30"`
- [ ] Message field accepts up to 500 chars.
- [ ]  All the fields must have the `required` attribute